EIPS/eip-7923.md

```diff
@@ -145,7 +145,10 @@ index bf34fbee..f63b7460 100644
 +        ALLOCATE_PAGE_COST = 100
 +        THRASH_PAGE_COST = 6
 +        LOWER_BITS = 12  # bits ignored for page calculations
-+        assert 2**LOWER_BITS == 4096
++        PAGE_SIZE = 4096
++        assert 2**LOWER_BITS == PAGE_SIZE   # sanity check
++        MAXIMUM_MEMORY_SIZE = 64 * 1024 * 1024
++        TRANSACTION_MAX_PAGES = MAXIMUM_MEMORY_SIZE // PAGE_SIZE
```

Comment:
Small style nit: these constants are now inside a method, I think these should be put outside the method.

Reply:
hmm, if this were intended for production code, i would agree with you, but since the style of the reference implementation here is as a patch, i think it is more readable for the spec / keeps the patch smaller for it to be close to where it's used.